Content.Shared/Cuffs/SharedCuffableSystem.cs

```diff
@@ -178,6 +179,21 @@ private void OnBeingPulledAttempt(EntityUid uid, CuffableComponent component, Be
                 args.Cancel();
         }
 
+        private void OnBuckleAttemptEvent(EntityUid uid, CuffableComponent component, ref BuckleAttemptEvent args)
+        {
+            // if someone else is doing it, let it pass.
+            if (args.UserEntity != uid)
+                return;
+
+            if (!TryComp<HandsComponent>(uid, out var hands) || component.CuffedHandCount != hands.Count)
+                return;
+
+            args.Cancelled = true;
+            var message = Loc.GetString("handcuff-component-cuff-interrupt-buckled-message");
+            if (_net.IsServer)
+                _popup.PopupEntity(message, uid, args.UserEntity);
```

Comment:
Isnt there a new networked popup variation

Reply:
you can't use it because this code gets called from the server for the alert, in which case it won't show up at all.